randovania/games/dread/json_data/Cataris.txt

```diff
@@ -30,7 +30,7 @@ Extra - asset_id: collision_camera_000
   > Event - Missile Tank Blob through wall (breakablemag002)
       Shoot Diffusion or Wave
   > Mid-Level
-      Simple IBJ or Use Spin Boost
+      Walljump (Beginner) or Simple IBJ or Use Spin Boost
```

Comment:
This can just be made trivial due to beginner wall jumps like this being treated as trivial now. This also goes for any other case in this PR.